File: src/main/java/com/wearby/wearby_api/dto/PrendaFiltroDTO.java
package com.wearby.wearby_api.dto;

import lombok.Data;

/**
 * DTO para transferir datos de prendas en el proceso
 * de generación de outfits. Evita exponer la entidad
 * completa y reduce el acoplamiento entre capas.
 */

@Data
public class PrendaFiltroDTO {
    private Integer id;
    private String nombre;
    private String imagenUrl;
    private Boolean favorito;
    private String categoria;
    private String estilo;
    private String temporada;
    private String formalidad;
    private String color;
}

File: werarbyAndroid/app/src/main/java/com/example/wearbyandroid/modelo/PrendaFiltroDTO.kt
package com.example.wearbyandroid.modelo

data class PrendaFiltroDTO(
    val id: Int,
    val nombre: String,
    val imagenUrl: String?,
    val favorito: Boolean,
    val categoria: String?,
    val color: String?,
    val estilo: String?,
    val temporada: String?,
    val formalidad: String?
)
